README.md

```diff
@@ -63,20 +69,24 @@ The basic parameters to fill are the genome name, the paths to the long and (QC'
 (The DB dir is a directory of your choosing where the databases and scripts will be stored.)
 
 Other very important parameters:
+
 - `flye_input_mode`: set to "nano-hq" if your reads have <= 5% error rate; otherwise use "nano-raw"
-- `medaka_model`: set this parameter to match the flow cell version / basecalling model you used to generate the long reads. See details in the [Medaka Github repo's "Models" section](https://github.com/nanoporetech/medaka#models)
-- "Post-polishing contig filter" section: set the min depth and coverage you would like for a contig to be kept (more info in that section of the YAML file)
+- `medaka_model`: set this parameter to match the flow cell version / basecalling model you used to generate the long
+  reads. See details in the [Medaka Github repo's "Models" section](https://github.com/nanoporetech/medaka#models)
+- "Post-polishing contig filter" section: set the min depth and coverage you would like for a contig to be kept (more
+  info in that section of the YAML file)
 
 There are other advanced parameters that you can also edit if you'd like.
 
 Lastly, make sure you set the threads and memory to values that make sense for your server.
 
 One you're finished editing the config file, save it as something like `myconfig.yaml`.
 
-**Note**: short read QC is not performed by _rotary_, so you'll want to do short read QC prior to using this pipeline. 
+**Note**: short read QC is not performed by _rotary_, so you'll want to do short read QC prior to using this pipeline.
 I recomend using the qc module of the [ATLAS pipeline](https://github.com/metagenome-atlas/atlas) for quick/robust QC.
 
 ### 3. Run
```

Comment:
Similarly to above, we'll need to update this section in future.

Reply:
Added as issue.